samples/bot-teams-catering/csharp/Manifest/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.7/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.7",
+    "version": "2.0.0",
+    "id": "be518d02-7ebc-4f26-8f72-284aa8a43349",
+    "packageName": "teams.catering.bot",
+    "developer": {
+        "name": "Microsoft Corp.",
+        "websiteUrl": "https://www.microsoft.com/",
+        "privacyUrl": "https://privacy.microsoft.com/en-US/privacystatement",
+        "termsOfUseUrl": "https://aka.ms/ActionsPlatform/Poll/TermsOfUse"
+    },
+    "icons": {
+        "color": "color.png",
+        "outline": "outline.png"
+    },
+    "name": {
+        "short": "Teams Catering Bot (dipja)",
+        "full": "Teams Catering Bot (dipja)"
+    },
+    "description": {
+        "short": "Catering bot helps you place orders using adaptive cards",
+        "full": "It uses adaptive cards to place food and drinks order"
+    },
+    "accentColor": "#FFFFFF",
+    "bots": [
+        {
+            "botId": "be518d02-7ebc-4f26-8f72-284aa8a43349",
```

Comment:
Could you please add placeholder here? <<Azure Bot ID>>

Reply:
done